src/enhance-props.ts

```diff
@@ -1,12 +1,17 @@
 import {propEnhancers} from './enhancers'
 import expandAliases from './expand-aliases'
-import * as styles from './styles'
 import * as cache from './cache'
+import * as styles from './styles'
+import {BoxProps} from './box-types'
 
+interface EnhancedPropsResult {
+  className: string
+  enhancedProps: Partial<BoxProps>
+}
 /**
  * Converts the CSS props to class names and inserts the styles.
  */
-export default function enhanceProps(rawProps) {
+export default function enhanceProps(rawProps: Partial<BoxProps>): EnhancedPropsResult {
```

Comment:
Is this available publicly? If so we may not want to modify the return.

Reply:
Looks like only used by box component when I search from repo and exports

Reply:
All the public exports are here: https://github.com/segmentio/ui-box/blob/8dabb1736486c6576673e1fedc30a1e40c186ac8/src/index.tsx